spyder/api/ipythonconsole.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+This module contains the ipython console API
+"""
+
+from spyder.plugins.ipythonconsole.utils.messagehandler import SpyderMessageHandler
+
+class IPythonAPIMixin:
```

Comment:
You need to inherit from ``object`` or this class will not behave properly in Python 2 (since it will be an obsolete "old style" class).